src/Symfony/Bundle/SecurityBundle/Logout/LogoutUrlGenerator.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Symfony\Bundle\SecurityBundle\Logout;
+
+use Symfony\Bundle\SecurityBundle\Security\FirewallMap;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+use Symfony\Component\Security\Http\FirewallMapInterface;
+use Symfony\Component\Security\Http\Logout\LogoutUrlGenerator as BaseLogoutUrlGenerator;
+
+class LogoutUrlGenerator extends BaseLogoutUrlGenerator
```

Comment:
Should we provide an interface and use decoration instead? IMHO it's not especially better here, but WDYT?